srcpkgs/wasmtime/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'wasmtime'
+pkgname=wasmtime
+version=0.19.0
+revision=1
+build_style=cargo
+short_desc="Standalone JIT-style runtime for WebAssembly, using Cranelift"
+maintainer="Isaac Freund <ifreund@ifreund.xyz>"
+license="Apache-2.0"
+homepage="https://github.com/bytecodealliance/wasmtime"
+_wasm_c_api_commit=d9a80099d496b5cdba6f3fe8fc77586e0e505ddc
+_WASI_commit=2b027d91c29e6141a3768e638076daa23f51621c
+distfiles="${homepage}/archive/v${version}.tar.gz
+ https://github.com/WebAssembly/wasm-c-api/archive/${_wasm_c_api_commit}.tar.gz
+ https://github.com/WebAssembly/WASI/archive/${_WASI_commit}.tar.gz"
+checksum="f569c74aacd4fe52c5fabe4fda9dd6c41bce72e8e467b52a38626bc0dbad1149
+ aea8cd095e9937f1e14f2c93e026317b197eb2345e7a817fe3932062eb7b792c
+ caf4343207d9f3ec135aae8c113824a8b588db61538d210b22acd6ea7c6d485e"
+
+case ${XBPS_TARGET_MACHINE} in
+	armv7l|armv6l-musl|aarch64-musl)
```

Comment:
i suspect: `armv* | aarch64* ` (and probably ppc as well, but that needs verification)